context/app/templates/pages/base_react.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="{{ url_for('static', filename='titles.css') }}">
+    {% block extra_stylesheets %}{% endblock %}
+
+    <title>{% block title_text %}{% endblock %} | HuBMAP</title>
+    <script>
+      let isLoggedIn = false;
+      {% if is_authenticated %}
+        isLoggedIn = true;
+      {% endif %}
```

Comment:
I would prefer to keep the same kinds of names across all levels, unless it really does mean something different. ... but `isLoggedIn` is probably the better name. Unless you want to fix this, file an issue, and we'll update the python code at some point.